packages-common/pluggable-widgets-tools/configs/webpack.config.common.js

```diff
@@ -15,41 +15,9 @@ const widgetConfig = {
         publicPath: "/"
     },
     resolve: {
-        extensions: [".ts", ".js", ".tsx", ".jsx"],
-        alias: {
-            tests: `${variables.projectPath}/tests`
-        }
-    },
-    module: {
```

Comment:
Can we somehow de-duplicate this code? A sloppy option would be to just put it back, since it is already conditional with `--hot` option. Otherwise we can make `common.js` to export a function that takes a parameter that defines whether to use hot reloading or not.

Actually parameter approach is more conventional for webpack - your config can always be a function that receives a `env` parameter. This way we can even merge our dev/prod/common configs into one with some conditionals in it.

Also following this path it would be more logical to have `--env.hot` parameter iso just `--hot`, since then webpack will do parsing for you.